luaui/Widgets/export_defs.lua

```diff
@@ -0,0 +1,60 @@
+function widget:GetInfo()
+	return {
+		name = "Unitdefs JSON Export",
+		desc = "Exports UnitDefs into JSON files.",
+		author = "Wereii",
+		date = "June 2023",
+		license = "GNU GPL, v2 or later, BA/BAR only",
```

Comment:
is saying that you apply GPL to BA/BAR only even possible? I believe stating that is just incompatible with GPL.

Reply:
I copied this from some other widget, I don't really care about the license.

Reply:
Then I suggest to change it to "GNU GPL, v2 or later" because I think that the "only" part is not compatible with GPL, so it doesn't make many sense.

Reply:
Sure, done